### ISSUE TITLE
upgrade thor dependency please (rails 6.1 upgrade blocker)

### ISSUE THREAD

**Issue by skcc321** [prontolabs/pronto#391]
Bundler could not find compatible versions for gem "thor":
```
  In Gemfile:
    pronto was resolved to 0.10.0, which depends on
      thor (~> 0.20.0)

    rails (= 6.1) was resolved to 6.1.0, which depends on
      railties (= 6.1.0) was resolved to 6.1.0, which depends on
        thor (~> 1.0)
```
please upgrade thor dependency because it blocks Rails upgrade.

**Comment by jer0m** [comment]
Hey @skcc321, I created a fork at https://github.com/jer0m/pronto that works with Rails 6.1. I also created a PR #393

**Comment by ashkulz** [comment]
Ah, I understand what the problem is. #370 already did what you want, but it isn't released yet (see #376). Please use the latest version from `git`.
